form-flow/src/main/kotlin/com/ritense/formflow/service/FormFlowService.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.formflow.service
+
+import com.ritense.formflow.domain.FormFlowDefinition
+import com.ritense.formflow.repository.FormFlowDefinitionRepository
+
+class FormFlowService(
+    private val formFlowDefinitionRepository: FormFlowDefinitionRepository
+) {
+
+    fun findLatestDefinitionByKey(formFlowKey: String): FormFlowDefinition? {
+        return formFlowDefinitionRepository.findFirstByIdKeyOrderByIdVersionDesc(formFlowKey)
+    }
+
+    fun save(formFlowDefinition: FormFlowDefinition) {
```

Comment:
I'd like to move this to the FormFlowDefinition in my own PR. No need to do anything with this right now, but I want to show some solutions that I think can be a lot cleaner to work with.